requirements.txt

```diff
@@ -4,5 +4,4 @@
 
 Adafruit-Blinka
 Adafruit-CircuitPython-miniMQTT
-Adafruit-CircuitPython-Requests
```

Comment:
Why is this removed? Don't we need requests?

Reply:
@brentru The examples need requests, but the library does not.